source/extensions/extensions_build_config.bzl

```diff
@@ -306,6 +306,13 @@ EXTENSIONS = {
 
     "envoy.retry_priorities.previous_priorities":       "//source/extensions/retry/priority/previous_priorities:config",
 
+    #
+    # Retry admission control
+    #
+
+    "envoy.retry_admission_control.static_limits": "//source/extensions/retry/admission_control/static_limits:config",
```

Comment:
This extension covers the `max_retries` field by providing a set limit to the number of concurrent retries per cluster and priority combination